index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portflio</title>
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+
+<body>
+    <div class="Mobile">
+        <ul class="header">
+            <li class="first"><a href="#">My Logo</a></li>
+            <li><a href="#">Portflio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+
+        <div class="free">
+
+        </div>
+
+
+        <div class="title">
+            <p>I’m Mariane
```

Comment:
As a best practice in semantic HTML, I suggest you use `h1` tag for this as it's the main focus of the webpage. 👍